Code/GraphMol/ROMol.cpp

```diff
@@ -437,7 +437,40 @@ unsigned int ROMol::addBond(Bond *bond_pin, bool takeOwnership) {
 }
 
 void ROMol::setStereoGroups(std::vector<StereoGroup> stereo_groups) {
-  d_stereo_groups = std::move(stereo_groups);
+  auto is_abs = [](const auto &sg) {
+    return sg.getGroupType() == StereoGroupType::STEREO_ABSOLUTE;
+  };
+
+  // if there's more than one ABS group, raise a warning and
+  // merge them
+  if (auto num_abs = std::ranges::count_if(stereo_groups, is_abs);
+      num_abs <= 1) {
+    d_stereo_groups = std::move(stereo_groups);
+  } else {
+    BOOST_LOG(rdWarningLog)
```

Comment:
I don't think you need the warning here.  If the RDKit spec is exactly one ABS group, then this shouldn't be a surprise hence no warning.